RELEASE_NOTES.md

```diff
@@ -8,6 +8,13 @@ This document contains release notes for the current major release and all patch
 For prior releases,
 see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
+## 12.17.5 (2017-10-24)
+
+This release fixes a regression.
+  - Fix for users on clouds where the ohai (v8.23.0) plugin for their cloud
+    doesn't report a list of strings for `public_ipv4` and `private_ipv4`.
```

Comment:
This is technically `public_ips` and `private_ips`.  I mislabeled them in chat.

I wonder if the technical details are helpful to users.  Maybe just:

> This release fixes a regression.

> - Set `nginx['use_implicit_hosts'] = false` by default.  This fixes a regression for users deploying on Digital Ocean and potentially other non-AWS cloud platforms where `nginx['use_implicit_hosts'] = true` results in incorrect nginx configuration.